homeassistant/components/netatmo/sensor.py

```diff
@@ -581,23 +477,32 @@ def __init__(
         """Initialize the sensor."""
         super().__init__(netatmo_device.data_handler)
         self.entity_description = NetatmoSensorEntityDescription(
```

Comment:
Side note: As this description is static we can define it as a constant at the module level.